src/gc/env/gcenv.base.h

```diff
@@ -41,8 +41,15 @@
 //
 
 typedef int BOOL;
+typedef uint8_t BYTE;
```

Comment:
These changes should not be needed now. Out ultimate goal is to eliminate use of the Windows-specific types from the GC.